### DISCUSSION TITLE
Exploring new designs for tags layout

### DISCUSSION THREAD

**Discussion by cyrus01337** [haotian2006/MyDocs#21] [Ideas]
At the moment, all tags are generated and sprawled out as buttons, though I'm not sure if this approach is really helpful:
![image](https://github.com/user-attachments/assets/8c8c878d-6c6c-4f8d-b8b5-334507688bb5)

I was thinking of re-implementing a global side-menu and listing the tags that way, and the main content would look similar to search engine results like below:
![image](https://github.com/user-attachments/assets/432ce849-e133-4d01-ac10-25a0aa5b67b0)

Thoughts? Ideas?

**Comment by haotian2006** [comment]
Yeah I do agree that the button ones don't look so good. And the search engine layout looks good

**Comment by haotian2006** [comment]
![image](https://github.com/user-attachments/assets/705bfa8e-1b0e-4a3e-a484-6c9a50e00bb4)
![image](https://github.com/user-attachments/assets/a6f5b077-762c-411e-85ac-b0353b81396f)

@cyrus01337 what do you think of this? anything I should add to make it look better since I feel like it looks a bit bland

**Reply by cyrus01337** [reply]
[This](https://play.tailwindcss.com/juicggdhW3) or something like it would suffice. If you're happy to add icon flairs relating to the subject then that can be included provided we have metadata tied to the post.

I don't recall if you've setup post metadata, it's been on my radar for a while now.

**Comment by haotian2006** [comment]
![image](https://github.com/user-attachments/assets/240474d5-b535-4dc2-b4f9-93ae9c9bf02c)
![image](https://github.com/user-attachments/assets/17986291-97a9-47d1-bbff-eb876222672c)

**Reply by cyrus01337** [reply]
Happy with the colours, see [here](https://github.com/haotian2006/RbxDocs/discussions/21#discussioncomment-10624527) for an alternative layout.

**Comment by haotian2006** [comment]
![Recording 2024-09-12 at 19 01 20](https://github.com/user-attachments/assets/1610e0d3-f67e-4abb-920b-cc7a624f4a9a)
@cyrus01337 this is what the alternative layout looks line

**Reply by cyrus01337** [reply]
Needs a smidge of space between the title and description. The outline does look basic though, I was aiming for a card-like design that you see [here](https://astro.build/) and [here](https://nextjs.org/).

**Reply by haotian2006** [reply]
Wait do you mean like have the tags as a grid layout or make them smaller

**Reply by cyrus01337** [reply]
A grid layout wouldn't be bad though I was thinking to match the design of the cards, which would be the following:
![Demonstration of cards from Astro's front page](https://github.com/user-attachments/assets/c49aa292-db9d-4272-90e9-dd19683e57dd)
![Demonstration of cards from NextJS's front page](https://github.com/user-attachments/assets/9eb2c328-93dc-4f9f-b6a9-01b852432634)

Hopefully that's clearer.

**Reply by haotian2006** [reply]
![image](https://github.com/user-attachments/assets/5b0d6a8c-15c3-431c-ad03-1bcde221c0c3)
Here is what I have rn

**Reply by cyrus01337** [reply]
Love it!
